test/integration/mongod/bats/mongod.bats

```diff
@@ -1,55 +0,0 @@
-#!/usr/bin/env bats
```

Comment:
We can have both in there. Having the serverspec won't hurt anything, but we use the bats tests for our testing.